templates/vue/src/components/SettingsModal.vue

```diff
@@ -109,6 +109,45 @@
               </b-col>
             </b-row>
           </b-form-group>
+          <b-form-group
+            label="World map"
+            description="Replace Tapestry with a new authoring experience over a map of the Earth"
+          >
+            <b-form-checkbox v-model="renderMap" switch>
+              {{ renderMap ? "Enabled" : "Disabled" }}
+            </b-form-checkbox>
+          </b-form-group>
+          <div v-if="renderMap">
+            <p v-if="!latitudeRangeValid" class="error-message">
+              North coordinate must be greater than south coordinate
+            </p>
+            <p v-if="!longitudeRangeValid" class="error-message">
+              East coordinate must be greater than west coordinate
+            </p>
```

Comment:
As mentioned before (but not yet done) please use Bootstrap-Vue provided functionality for showing errors. See https://bootstrap-vue.org/docs/components/form-input#contextual-states